### ISSUE TITLE
Edges need to have coordinates as an endpoint instead of only Nodes.

### ISSUE THREAD

**Issue by liamross** [liamross/psiagram#45]
**Describe the bug**
Currently, edges need to have a Node as their end points. They must also be able to have coordinates.

**Expected behavior**
An edge with ICoordinates as an endpoint should render.

**Comment by liamross** [comment]
Properties in the Edge:

```js
// To attach to a Node:
source = { id: 'example-node-1' };
target = { id: 'example-node-2' };

// To attach to a coordinate point:
source = { x: 80, y: 120 };
target = { x: 140, y: 180 };
```

To change:

1.
https://github.com/liamross/psiagram/blob/137eb45481028101cc41a31eb6c7a49e538cc7bf/packages/psiagram/src/components/Paper/Paper.base.ts#L372-L373

2. 
https://github.com/liamross/psiagram/blob/137eb45481028101cc41a31eb6c7a49e538cc7bf/packages/psiagram/src/components/Paper/Paper.base.ts#L390

3. 
https://github.com/liamross/psiagram/blob/137eb45481028101cc41a31eb6c7a49e538cc7bf/packages/psiagram/src/components/Paper/Paper.base.ts#L470-L515

**Comment by liamross** [comment]
Fixed by #55
